components/event-feed-service/pkg/server/feed.go

```diff
@@ -85,6 +85,11 @@ func (f *FeedService) GetFeedSummary(countCategory string, filters []string,
 		return nil, err
 	}
 
+	mapFilters, err = filterByProjects(ctx, mapFilters)
+	if err != nil {
+		return nil, err
+	}
+
```

Comment:
These five lines implements the change. Over 600 lines of test for 5 lines :sweat_smile: